my-app/src/components/ContentList.jsx

```diff
@@ -14,9 +16,9 @@ const ContentList = ({ content }) => {
               <h4 className="content_title">{content.title} </h4>
               <p> {content.description.substring(0, 200)}... </p>
               {/* 
-             todo : add link instead of the anchor...
+             todo : pass the data of a specific news from <ContentList /> to <NewsContent />
              */}
-              <a href={content.url}>Read More </a>
+              <Link to="/NewsContent" >Read More </Link>
```

Comment:
In order to catch each new details, you need to pass the specific `${id}` of it too, 
```jsx
<Link to={`/news-content/${content.source.id}`} >....
```
where this endpoint is already attached to `NewsContent` Route 
```jsx
<Route path="/news-content/:id" component= { NewsContent } />
```